hbase-shell/src/main/ruby/hbase/admin.rb

```diff
@@ -537,8 +537,8 @@ def assign(region_name)
 
     #----------------------------------------------------------------------------------------------
     # Unassign a region
-    def unassign(region_name, force)
-      @admin.unassign(region_name.to_java_bytes, java.lang.Boolean.valueOf(force))
+    def unassign(region_name)
```

Comment:
this breaks compatibility for the shell and will need to be release noted. Alternatively you could do the analagous thing as in the Admin apis and deprecate the second parameter by making it optional and stating it is ignored.

Reply:
Will fix later. Thanks.

Reply:
this compatibility break is still present. could you correct it similar to how you did for the unassign shell command?

Reply:
Fixed, Thanks.